README.md

```diff
@@ -30,79 +28,17 @@ This is a proof-of-concept application which generates a customized looking queu
-<li><a href="https://instant.webex.com/" target="_blank">Webex Instant Connect</a> - To embed Webex IC flow into the support flow, Instant Connect must be activated</li>
-<li><a href="https://cisco.sharepoint.com/:b:/s/WXSD-WebexSolutionsDevelopment/EQJu96KU411LpGw16KdT4ewBwZsAOl7NEH_Tnprt6UX4tA" target="_blank">Webex Calling Queue on Cisco Devices</a> - To create a support queue experience on Cisco Device using macros & Webex calling group call management</li>
-</ol>
-
-## Browser Meeting SDK
-
-### Requester Flow
-
-Establishes a WebSocket connection to SoapBox to publish the request data and subscribe to queue & meeting management events.
-
-### Responder Flow
-
-<ol>
-<li>Establish a WebSocket connection to SoapBox to register queue events</li>
-<li>Send HTTPs POST request to Mindy BOT</li>
-<li><a href="https://github.com/wxsd-sales/mindy-bot-refactored" target="_blank" >Mindy Bot</a> Node Service
-   <ol>
-      <li> Issues JWTs</li>
-      <li> Create an Space</li>
-      <li> Create Space memberships with newly generated guest tokens</li>
-      <li> Retrieve a meeting link using <a href="https://github.com/wxsd-sales/wxsd-guest-demo" target="_blank" >Guest Demo</a> API</li>
-      <li> Update registered soapbox session with meeting details</li>
-   </ol>
-</li>
-<li> Responder and requester clients receive the meeting information from the SoapBox
-<li> Responder Client initiates Browser Meeting SDK to register Meeting related events </li>
-<li> Responder Client publishes queue <strong>pop</strong> & meeting related events back to requester client </li>
-</li>
-</ol>
-<p align="center">
-   <img src="static/readme/SDK.jpg" alt="video-call-queue"/>
-</p>
-
-<br />
-
-## Instant Connect Solution
-
-### Requester Flow
-
-Establishes a WebSocket connection to SoapBox to publish the request data and subscribe to queue & meeting management events.
-
-### Responder Flow
-
-<ol>
-<li> Establishes a WebSocket connection to SoapBox to register queue related events</li>
-<li> Sends HTTPs POST request Instant Connect</li>
-<li> Receives IC meeting url to join </li>
-<li> Responder Client initiates Browser Meeting SDK to register Meeting related events </li>
-<li> Responder Client publishes queue <strong>pop</strong> & meeting related events back to requester client </li>
-</li>
-</ol>
-<p align="center">
-   <img src="static/readme/IC.jpg" alt="video-call-queue"/>
-</p>
-
-<br />
+This PoC leverages a Webex Group Call Management feature powered by Webex Cloud Calling behind the scene to provide a FIFO orderly queue system to manage requests and establish a SIP connections between the responder and requester. Here is the list of services that are required before launching the application. For more information please click [here](https://cisco.sharepoint.com/:b:/s/WXSD-WebexSolutionsDevelopment/EQJu96KU411LpGw16KdT4ewBwZsAOl7NEH_Tnprt6UX4tA)
```

Comment:
We can't be 100% sure its a FIFO queue since the queue creation is not a part of the application; better not to list it as such.